src/SASjs.ts

```diff
@@ -167,6 +167,20 @@ export default class SASjs {
     return await this.sasViyaApiClient!.deleteContext(contextName, accessToken)
   }
 
+  /**
+   * Returns a compute context.
+   * @param contextName - the name of the context to return.
```

Comment:
same comment as above.  Also given that this has no parallel in SAS 9 - perhaps we should be more explicit, eg:

`getViyaComputeContextByName()`

and again - what is returned?  It would be nice to have an example in the documenation.

Reply:
ignore the viya bit - I see we don't do that for any other functions.  But we should change to 

`getComputeContextByName` - there are two types of context (Launcher Context and Compute Context) so we need to be explicit.